src/resources/InsightPanelInterfaces/InsightPanelInterface.ts

```diff
@@ -38,4 +44,34 @@ export default class InsightPanelInterface extends Resource {
             body,
         );
     }
+
+    listVersions(insightPanelInterfaceId: string, options: ListHostedInterfaceVersionsParams) {
+        return this.api.get<PageModel<InsightPanelInterfaceVersion>>(
+            this.buildPath(`${InsightPanelInterface.baseUrl}/${insightPanelInterfaceId}/versions`, options),
+        );
+    }
+
+    getVersion(insightPanelInterfaceId: string, versionNumber: number) {
+        return this.api.get<InsightPanelInterfaceVersion>(
+            `${InsightPanelInterface.baseUrl}/${insightPanelInterfaceId}/versions/${versionNumber}`,
+        );
+    }
+
+    restoreVersion(insightPanelInterfaceId: string, versionNumber: number, params: RestoreInterfaceVersionParams) {
```

Comment:
For consistency with the `updateVersionLabel`, I'd suggest to rename `params` to `body`.